src/main/java/com/epam/novostroinyi/core/assertion/Assert.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.novostroinyi.core.assertion;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Collection;
+
+public class Assert {
```

Comment:
What the purpose of this class? You can use assertions directly in code

Reply:
@IlliaKosholapenko The main idea here is to low coupling. It allows us to change assertions library with less efforts and even use several assertions libraries if it needed